### PR TITLE
Add shipping_rates api

### PR DESCRIPTION
**Pull request by nemonweb**

### CODE REVIEW

File: source/includes/_delivery_windows.md.erb [hunk 1]
```diff
@@ -34,3 +34,53 @@ curl "<%= config[:host] %>/shipments/4/delivery_windows?date=2016-01-01" \
 SHIPMENT_ID | Да | id груза
 date | Нет | Дата доставки
 
+
+## Получение окон для груза (новая механика ценообразования)
```
Comment (oleglitvin) [inline source/includes/_delivery_windows.md.erb]: `груза` --> `подзаказа`

File: source/includes/_delivery_windows.md.erb [hunk 2]
```diff
@@ -34,3 +34,53 @@ curl "<%= config[:host] %>/shipments/4/delivery_windows?date=2016-01-01" \
...
+        "id": 3,
+        "starts_at": "2018-10-02T09:00:00.000+03:00",
+        "ends_at": "2018-10-02T11:00:00.000+03:00"
+      }
+    }
+  ]
+}
+```
+
+Чтобы получить окна доставки для груза, выполните следующий запрос:
```
Comment (oleglitvin) [inline source/includes/_delivery_windows.md.erb]: подзаказа

File: source/includes/_delivery_windows.md.erb [hunk 3]
```diff
@@ -34,3 +34,53 @@ curl "<%= config[:host] %>/shipments/4/delivery_windows?date=2016-01-01" \
...
+
+`GET <% config[:host] %>/shipments/#{NUMBER}/shipping_rates`
+
+Данный запрос вернет все доступные окна доставки.
```
Comment (oleglitvin) [inline source/includes/_delivery_windows.md.erb]: It will return intervals for a one day

File: source/includes/_delivery_windows.md.erb [hunk 4]
```diff
@@ -34,3 +34,53 @@ curl "<%= config[:host] %>/shipments/4/delivery_windows?date=2016-01-01" \
...
+--------- | ------- | -----------
+NUMBER | Да | Номер доставки
+date | Нет | Дата доставки
+
+### Структура объекта ShippingRates
+
+Атрибут | Описание
+--------- | -----------
+total_value | Конечная стоимость доставки (с наценкой)
+value | Наценка. Может быть положительной, отрицательной или 0
```
Comment (oleglitvin) [inline source/includes/_delivery_windows.md.erb]: It's a cost without surge

**Comment by nemonweb** [comment]
@oleglitvin fixed

**Pull request merged**